### PR TITLE
Validator

### PR DESCRIPTION
**Pull request by apo-bozdag**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,26 @@ Available locales are `de`, `en` & `tr`
 </script>
 ```
 
+#### Validation
+
+There a 2 difference worker concerning how validation would interact with user input:
```
Comment (Orkuncakilkaya) [inline README.md:109]: Fix typos, There `are` 2 `different` `workers` concerning how validation wold interact with user input:

File: src/locales/en.json [hunk 2]
```diff
@@ -13,5 +13,16 @@
     "expiryMonth": "XX",
     "expiryYear": "XX",
     "number": "0000 0000 0000 0000"
+  },
+  "validation": {
+    "required": "Required.",
+    "cardFullName": "Your Name and Surname are Required.",
+    "cardNumber": "Invalid Credit Card Number.",
+    "cardExpiry": "Invalid Expiration Date.",
+    "cardCvv": "Invalid CVC.",
```
Comment (Orkuncakilkaya) [inline src/locales/en.json:22]: Typo, change `CVC to CVV`

File: src/locales/tr.json [hunk 3]
```diff
@@ -13,5 +13,11 @@
     "expiryMonth": "XX",
     "expiryYear": "XX",
     "number": "0000 0000 0000 0000"
+  },
+  "validation": {
+    "cardFullName": "İsim ve Soyisminiz Gereklidir.",
+    "cardNumber": "Geçersiz Kredi Kartı Numarası.",
+    "cardExpiry": "Geçersiz Sona Erme Tarihi.",
+    "cardCvv": "Geçersiz CVC."
```
Comment (Orkuncakilkaya) [inline src/locales/tr.json:21]: Typo, change `CVC` to `CVV`

File: src/validation/templates.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  required: "Required.",
+  cardNumber: 'Invalid Credit Card Number.',
+  cardExpiry: "Invalid Expiration Date.",
+  cardCvv: "Invalid CVC.",
```
Comment (Orkuncakilkaya) [inline src/validation/templates.js:7]: Typo, change `CVC` to `CVV`

**Comment by apo-bozdag** [comment]
test

**Comment by Orkuncakilkaya** [comment]
Fracto team will work on a Validator patch soon, you'll be credited for Validator but this patch is not going to put on use. We appreciate your work thanks! 👍

**Pull request closed without merging**